Tasks/NodeToolV0/task.json

```diff
@@ -39,6 +39,22 @@
             "defaultValue": "false",
             "required": false,
             "helpMarkDown": "Always checks online for the latest available version that satisfies the version spec. This is typically false unless you have a specific scenario to always get latest. This will cause it to incur download costs when potentially not necessary, especially with the hosted build pool."
+        },
+        {
+            "name": "arch",
+            "type": "pickList",
+            "label": "Architecture",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The target architecture (x86, x64, ARM) of the Node install.",
+            "groupName": "advanced",
+            "options": {
+                "x86": "x86",
+                "x64": "x64",
+                "ARM64": "arm64",
+                "ARMv6": "armv61",
+                "ARMv7": "armv71"
```

Comment:
Hi, tiny typos here, I believe:

numeral `1` should be lowercase L `l` in the ARM arch option strings

e.g.:
- https://nodejs.org/dist/latest-v10.x/
  - https://nodejs.org/dist/latest-v10.x/node-v10.21.0-linux-armv6l.tar.gz
  - https://nodejs.org/dist/latest-v10.x/node-v10.21.0-linux-armv7l.tar.gz